tests/integration/pure_cairo_vm/cheatcodes/expect_call/expect_call_test.cairo

```diff
@@ -0,0 +1,64 @@
+from starkware.cairo.common.math import assert_not_zero
```

Comment:
Can we add a test for proxy calling the contract? This should also be registered as a contract call.

Reply:
I think this is blocked by https://github.com/software-mansion/protostar/issues/1361

I'm going to add a TODO to this PR with the proxy tests

Reply:
Added as TODO in the main PR comment, resolving this one